build.gradle

```diff
@@ -1038,6 +1048,16 @@ project('test:system') {
     task startK8SystemTests(type: Test) {
         dependsOn 'copyTestsToK8sCluster', 'runFluentBitInstall' // ensure test artifacts are copied to the cluster.
         description 'Used to invoke K8s based System Tests, example usage: gradle startK8SystemTests'
+
+        def excludedTests = System.getProperty('excludeSystemTests')
```

Comment:
Wondering if we could have this code block in a function rather than repeat it twice in the same file.